src/apis/rental/useRentalRequest.js

```diff
@@ -0,0 +1,22 @@
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+
+const useRentalRequest = () => {
+  const navigate = useNavigate();
+
+  const submitRental = async (data) => {
```

Comment:
```suggestion
  const submitRental = (data) => {
```
이제는 async를 사용하지 않아도 됩니다.